README.md

```diff
@@ -0,0 +1,105 @@
+# Vault HA Cluster
+
+This describes how to deploy a Vault HA cluster.
+
+## Prerequisites
+
+### System
+
+The [Terraform](https://www.terraform.io/) application must be installed on the system, as well as the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/).
```

Comment:
Info about cloning the repo?

Reply:
Not needed, peoples who need the repo should know how to use git/github